setup.py

```diff
@@ -1,12 +1,68 @@
-import setuptools  # type: ignore
+import os
 
+import numpy as np
+import setuptools
+from Cython.Build import cythonize  # type: ignore
 
+
+SOURCES = [
+    os.path.join("spokestack/extensions/webrtc", source)
+    for source in [
+        "filter_audio/other/complex_bit_reverse.c",
+        "filter_audio/other/complex_fft.c",
+        "filter_audio/other/copy_set_operations.c",
+        "filter_audio/other/cross_correlation.c",
+        "filter_audio/other/division_operations.c",
+        "filter_audio/other/dot_product_with_scale.c",
+        "filter_audio/other/downsample_fast.c",
+        "filter_audio/other/energy.c",
+        "filter_audio/other/get_scaling_square.c",
+        "filter_audio/other/min_max_operations.c",
+        "filter_audio/other/real_fft.c",
+        "filter_audio/other/resample_by_2.c",
+        "filter_audio/other/resample_by_2_internal.c",
+        "filter_audio/other/resample_fractional.c",
+        "filter_audio/other/resample_48khz.c",
+        "filter_audio/other/spl_init.c",
+        "filter_audio/other/spl_sqrt.c",
+        "filter_audio/other/spl_sqrt_floor.c",
+        "filter_audio/other/vector_scaling_operations.c",
+        "filter_audio/vad/vad_core.c",
+        "filter_audio/vad/vad_filterbank.c",
+        "filter_audio/vad/vad_gmm.c",
+        "filter_audio/vad/vad_sp.c",
+        "filter_audio/vad/webrtc_vad.c",
+        "filter_audio/agc/analog_agc.c",
+        "filter_audio/agc/digital_agc.c",
+        "filter_audio/ns/nsx_core.c",
+        "filter_audio/ns/nsx_core_c.c",
+        "filter_audio/ns/noise_suppression_x.c",
+    ]
+]
+
+EXTENSIONS = [
```

Comment:
We have a bit of an inconsistency in naming of the webrtc stuff now from the client's perspective. For each of the webrtc components, this is what the client would have to import.

```
spokestack.extensions.webrtc.agc.AutomaticGainControl
spokestack.extensions.webrtc.ans.AutomaticNoiseSuppression
spokestack.vad.webrtc.VoiceActivityDetector/VoiceActivityTrigger
```

The client doesn't really care about the term `extension`, so I would recommend making new modules/classes `spokestack.agc.webrtc.AutomaticGainControl` and `spokestack.nsx.webrtc.AutomaticNoiseSuppression` if you can get them to build like that. If not, see if you can re-export those classes under these module names instead of the `extension` module names.